bank_statement_reader.py

```diff
@@ -0,0 +1,15 @@
+from bank_statement_reader.BankStatementExecutor import BankStatementExecutor
+
+executor = BankStatementExecutor()
+print(executor.BANK_STATEMENTS_CHOICES)
```

Comment:
Do we want to remove this prints?

Reply:
This is for testing.

Reply:
This file is not executed in production, it is for testing bank statements individually and locally.